src/main/java/controller/UserController.java

```diff
@@ -0,0 +1,51 @@
+package controller;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.User;
+import model.UserResponseDto;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import service.UserService;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @GetMapping("/get/{id}")
+    public UserResponseDto get(@PathVariable Long id) {
+        User user = userService.get(id);
+        UserResponseDto userResponseDto = new UserResponseDto();
+        userResponseDto.setEmail(user.getEmail());
+        userResponseDto.setName(user.getName());
+        return userResponseDto;
+    }
+
+    @GetMapping("/getall")
```

Comment:
to get one user url would be user/2 where  2 is userId. So in case of getting all users we can use just user/
```suggestion
    @GetMapping
```

Reply:
Done